<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>406</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>316</width>
     <height>386</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>316</width>
     <height>386</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QFrame {
	background: rgb(186, 189, 182);
    border-style: solid;
    border-color: rgb(255, 255, 255);
    border-width: 1px;
    border-radius: 6px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>296</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame {
	background: rgb(186, 189, 182);
    border-style: solid;
    border-color: rgba(255, 255, 255, 155);
    border-width: 1px;
    border-radius: 6px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="IntLineEdit" name="tool_number_input">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>5</y>
       <width>45</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="default_value" stdset="0">
      <number>0</number>
     </property>
     <property name="bottom" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="tool_number_label_3">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>5</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Tool Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="tool_description">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>35</y>
       <width>246</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans Narrow</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>No Tool Selected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_171">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>95</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>rpm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_170">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>95</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Spindle Speed:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="FloatLineEdit" name="spindle_rpm_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>95</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>33</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="default_value" stdset="0">
     <double>100.000000000000000</double>
    </property>
    <property name="format_string" stdset="0">
     <string>.0f</string>
    </property>
    <property name="bottom" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_179">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>325</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_174">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>279</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Z Feed Rate:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="FloatLineEdit" name="clearance_height_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>325</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>33</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="format_string" stdset="0">
     <string>%.3f</string>
    </property>
    <property name="bottom" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_172">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>185</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Coolant:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_173">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>235</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>XY Feed Rate:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_176">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>235</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>mm/min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_177">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>280</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>mm/min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="spindle_direction_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>140</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_178">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>140</y>
      <width>86</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Direction:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="FloatLineEdit" name="xy_feed_rate_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>235</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>33</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="format_string" stdset="0">
     <string>%.1f</string>
    </property>
    <property name="bottom" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_175">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>325</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Clearance Height:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="coolant_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>185</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="FloatLineEdit" name="z_feed_rate_input">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>279</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>33</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    border-style: transparant;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 3px;
    color: black;
    background: rgb(235, 235, 235);
    padding-right: 2px;
	font-family: &quot;Liberation Sans Narrow&quot;;
    font-size: 13pt;
}</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="format_string" stdset="0">
     <string>%.3f</string>
    </property>
    <property name="bottom" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntLineEdit</class>
   <extends>VCPLineEdit</extends>
   <header>widgets.conversational.int_line_edit</header>
  </customwidget>
  <customwidget>
   <class>FloatLineEdit</class>
   <extends>VCPLineEdit</extends>
   <header>widgets.conversational.float_line_edit</header>
  </customwidget>
  <customwidget>
   <class>VCPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qtpyvcp.widgets.input_widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
